examples/get-started/codelab_web/lib/starter.dart

```diff
@@ -47,7 +47,7 @@ class _SignUpFormState extends State<SignUpForm> {
   final _lastNameTextController = TextEditingController();
   final _usernameTextController = TextEditingController();
 
-  double _formProgress = 0;
+  final double _formProgress = 0;
```

Comment:
I'm curious why this is final. Going by the name, `_formProgress` feels like it should be updated during the progress of the form?

Reply:
Great observation. I modified it because the linter raises warnings, I'll take another look.